### ISSUE TITLE
Custom time/date variable

### ISSUE THREAD

**Issue by nmalcolm** [gophish/gophish#194]
Hi,

It would be great if there was a customizable time/date variable. An example where this would be useful is an email template for a new account sign in. It would make templates look more legit and would increase the chance of the target taking immediate action.

Thanks.

**Comment by jordan-wright** [comment]
I bet I can make something like this work. Let me see what I can do :smile: 

Any particular format you'd want it in?

**Comment by nmalcolm** [comment]
As different services use different formats, the ability to use a custom format for each template would be super useful. I was thinking something like how PHP's date() function works.

Would be that possible?

**Comment by securitygeneration** [comment]
Agreed this would be a nice one to have (but I do agree we'd need the ability to set a custom format)
